app/i18n/locales/en-US.json

```diff
@@ -226,5 +229,39 @@
   "wallet.transaction.transactionId": "Transaction ID",
   "wallet.transaction.type": "{currency} transaction",
   "wallet.transaction.type.card": "Card payment",
-  "wallet.transaction.type.exchange": "Exchange"
-}
\ No newline at end of file
+  "wallet.transaction.type.exchange": "Exchange",
+  "wallet.trezor.dialog.title.label": "CONNECT TO TREZOR HARDWARE WALLET",
+  "wallet.trezor.dialog.trezor.common.step.link.helpYoroiWithTrezor": "https://yoroi-wallet.com/",
```

Comment:
![image](https://user-images.githubusercontent.com/19986226/49360809-98889400-f71d-11e8-95ea-abfe3f2b3be2.png)

middle `trezor` is redundant

Reply:
DONE